docs/en/01_Working_with_projects/03_Recipes_and_supported_modules.md

```diff
@@ -41,8 +40,8 @@ The reasons include:
 * your site may break when infrastructure changes are rolled out - these changes will only be tested against the official CWP recipe releases
 
 If after careful consideration your development team decides to depart from the recipe model, it is recommended to pull
-in at least the [cwp](https://gitlab.cwp.govt.nz/cwp/cwp) module. If even this doesn't work for you, the last resort is
-including the [cwp-core](https://gitlab.cwp.govt.nz/cwp/cwp-core) module. This at least will ensure the minimal compatibility
+in at least the [cwp](https://github.com/silverstripe/cwp) module. If even this doesn't work for you, the last resort is
+including the [cwp-core](https://github.com/silverstripe/cwp-core) module. This at least will ensure the minimal compatibility
```

Comment:
I think we should suggest recipes here instead of the modules themselves:

```diff
- the [cwp](https://github.com/silverstripe/cwp) module
+ the [CWP CMS recipe](https://github.com/silverstripe/cwp-recipe-cms)
- the [cwp-core](https://github.com/silverstripe/cwp-core) module
+ the [CWP core recipe](https://github.com/silverstripe/cwp-recipe-core)
```